docs/book/v1/authorization-server.md

```diff
@@ -20,6 +21,8 @@ use Mezzio\Authentication\OAuth2;
 
 $app->post('/oauth2/token', OAuth2\TokenEndpointHandler::class);
 ```
+## Parsing JSON payloads in the token endpoint
+To enable the token endpoint to handle the POST requests in JSON, the [Body Parsing Middleware](https://docs.mezzio.dev/mezzio/v3/features/helpers/body-parse/) helper should be included in the application. 
```

Comment:
If the middleware is required for the entire process then use "must" instead of "should":

> …helper **MUST** be included in the application.